charts/primary-site/templates/services/stream.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: stream
+  annotations:
+    {{- range $key, $value := .service.annotations }}
```

Comment:
the lint failure is because this .service.annotations needs to be scoped to streamService. Either in-line or using a "with" block at the top of this file like we have in some other cases.

Reply:
(The `ingress.yaml` works because there's a `{{- with .Values.ingress }}` on top — without that we need the full `{{- range $key, $value := .Values.streamService.service.annotations }}` here.)